TheraLang.BLL/Services/DataSeedService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Common.Helpers.PasswordHelper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TheraLang.BLL.Interfaces;
+using TheraLang.DAL.Entities;
+using TheraLang.DAL.UnitOfWork;
+
+namespace TheraLang.BLL.Services
+{
+    public class DataSeedService : IDataSeedService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        public DataSeedService(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task CreateDefaultData()
+        {
+            if (await _unitOfWork.Repository<Role>().Get(r => r.Name == "Admin") == null)
+            {
+                var Admin = new Role()
+                {
+                    Name = "Admin",
+                };
+                _unitOfWork.Repository<Role>().Add(Admin);
+                await _unitOfWork.SaveChangesAsync();
+            }
+            if (await _unitOfWork.Repository<Role>().Get(r => r.Name == "Member") == null)
+            {
+                var Member = new Role()
+                {
+                    Name = "Member"
+                };
+                _unitOfWork.Repository<Role>().Add(Member);
+                await _unitOfWork.SaveChangesAsync();
+            }
+            if (await _unitOfWork.Repository<Role>().Get(r => r.Name == "Guest") == null)
+            {
+                var Guest = new Role()
+                {
+                    Name = "Guest"
+                };
+                _unitOfWork.Repository<Role>().Add(Guest);
+                await _unitOfWork.SaveChangesAsync();
+            }
+            if (await _unitOfWork.Repository<User>().Get(u => u.Email == "admin") == null)
+            {
+                var adminRole = await _unitOfWork.Repository<Role>().Get(x => x.Name == "Admin");
+                var Admin = new User()
+                {
+                    Email = "admin",
```

Comment:
Shouldn't email field be something like that "admin@uttmm.com", cause you probably would have some validator, which wouldn't allow you to enter such string in email field?

Reply:
I created regex for it on another branch, where I work with regisrtation, but, as we use this users only for testing, it won't be very comfortable to write "admin@gmail ... " every time

Reply:
It hasn't real admin yet